earthfile2llb/interpreter.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/earthly/earthly/ast/hint"
 	"github.com/earthly/earthly/ast/spec"
 	"github.com/earthly/earthly/buildcontext"
+	"github.com/earthly/earthly/cmd/earthly/version"
```

Comment:
we shouldn't be importing any `cmd` packages from non-cmd packages -- this is to avoid getting into circular dependencies.

In this specific case `version` is definitely tied to the cmd, but it still feels a bit odd -- how about renaming the `github.com/earthly/earthly/cmd/earthly/version` package to `github.com/earthly/earthly/internal/version`.